src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier ballColor;
+    private Random ballNumber;
```

Comment:
```suggestion
    private ColorSupplier colorSupplier;
    private Random random;
```

Reply:
Is it important to name objects of a class with the same name as the class?